### PR TITLE
#41 Base frontend and backend for style assessment test

### PR DESCRIPTION
**Pull request by AndriyCh1**
Currently it has simple quiz components render and returns from backend only specific quiz category - work style assessment

### CODE REVIEW

**Review by deimonbody — APPROVED** [review]

**Review by nikita-remeslov — COMMENTED** [review]
Frontend looks good, waiting for review from @aleksandr-kuharenko

File: frontend/src/components/work-quiz/work-quiz-item/styles.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  border: none;
-}
-
 .test-item__checkbox {
   border-color: #463da8 !important;
   cursor: pointer;
 }
+
+.test-item__checkbox:checked {
+  background-color: #463da8 !important;
```
Comment (nikita-remeslov) [inline frontend/src/components/work-quiz/work-quiz-item/styles.scss]: color from `asset/scss/utils`

File: frontend/src/components/work-quiz/work-quiz-item/styles.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.test-item__checkbox:checked {
-  background-color: #463da8 !important;
-  border: none;
-}
-
 .test-item__checkbox {
   border-color: #463da8 !important;
```
Comment (nikita-remeslov) [inline frontend/src/components/work-quiz/work-quiz-item/styles.scss]: the same

File: frontend/src/components/work-quiz/styles.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.test-submit {
+  background-color: #463da8 !important;
```
Comment (nikita-remeslov) [inline frontend/src/components/work-quiz/styles.scss]: assets/scss/utils

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**